src/components/Transactions/Transactions.js

```diff
@@ -56,115 +61,120 @@ const TransactionsSkeleton = () => (
   </Card>
 );
 
-const Transactions = ({
-  transactions,
-  toggleTransactionsList,
-  isTransactionsItemsLoading,
-  isExpanded,
-  isShowMoreButtonHidden,
-  error,
-  isTransactionsSectionLoading,
-  t
-}) =>
-  isTransactionsSectionLoading ? (
-    <TransactionsSkeleton />
-  ) : (
-    <WrapStyled>
-      {error.length !== 0 ? (
+const Transactions = ({ t }) => {
+  const {
+    transactions: transactionsList,
+    showToggleButton,
+    error: transactionsError,
+    loading: isTransactionsSectionLoading,
+    isListExpanded: isTransactionListExpanded
```

Comment:
there is no other list than 'transactionList' so I don't see a point to rename it. Same for other state keys